### PR TITLE
Template support

### PR DESCRIPTION
**Pull request by Jan11979**

### CODE REVIEW

**Review by Flooooooooooorian — APPROVED** [review]

File: frontend/src/page/FrameSet.tsx [hunk 1]
```diff
@@ -43,15 +43,18 @@ function FrameSet() {
...
     const [rgbItem, setRGBItem] = useState(tmprgb);
 
-
+    const [reloadFixtureList, setReloadFixtureList] = React.useState("NO");
     let tmpActiveFixtureList: ActiveFixtureList[] = [];
     const [activeFixtureList, setActiveFixtureList] = useState(tmpActiveFixtureList);
     let tmpActiveFixtureListSelected: string[] = [];
     const [activeFixtureListSelected, setActiveFixtureListSelected] = useState(tmpActiveFixtureListSelected);
     useEffect(() => {
+        if(reloadFixtureList !== "NO"){
```
[the remark this answers is not in the document]
Comment (Jan11979) [reply frontend/src/page/FrameSet.tsx:52]: I call the reloadFixtureState in an other part of the Frontend.

File: frontend/src/controller/Fetching.ts [hunk 2]
```diff
@@ -33,6 +33,25 @@ export async function getActiveFixtureList() {
     return rawResponse.json();
 }
 
+
+export async function getActiveFixtureTemplateList() {
+    const rawResponse = await fetch(`/api/fixture/allfixturetemplatelist`, {
+        method: 'GET',
+        headers: {"Authorization": "Bearer" + "No Token"},
```
[the remark this answers is not in the document]
Comment (Jan11979) [reply frontend/src/controller/Fetching.ts:40]: The Token is and Security should be implemented this week

File: frontend/src/page/FrameSet.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Flooooooooooorian) [reply frontend/src/page/FrameSet.tsx:52]: Yeah but why dont you the function getActiveFixtureList directly in the other part?

[the remark this answers is not in the document]
Comment (Jan11979) [reply frontend/src/page/FrameSet.tsx:52]: i tried, but then the pageReload did not work

**Pull request merged**